app/models/manageiq/providers/openstack/builder.rb

```diff
@@ -4,6 +4,14 @@ def build_inventory(ems, target)
       case target
       when ManageIQ::Providers::Openstack::CloudManager
         cloud_manager_inventory(ems, target)
+      when ManageIQ::Providers::Openstack::StorageManager::CinderManager
+        inventory(
+          ems,
+          target,
+          ManageIQ::Providers::Openstack::Inventory::Collector::StorageManager::CinderManager,
+          ManageIQ::Providers::Openstack::Inventory::Persister::StorageManager::CinderManager,
+          [ManageIQ::Providers::Openstack::Inventory::Parser::StorageManager::CinderManager]
```

Comment:
@mansam for your question below, you need to ```ManageIQ::Providers::Openstack::Inventory::Parser::StorageManager::CinderManager```

to line 30, so ManagerRefresh::TargetCollection (targeted refresh) invokes all parsers